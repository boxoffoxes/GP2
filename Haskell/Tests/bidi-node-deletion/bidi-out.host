[
	(n1, empty)
	(n2, empty)
	(n3, empty)
|
	(e1, n2, n1, empty)
	(e2, n2, n3, empty)
]
