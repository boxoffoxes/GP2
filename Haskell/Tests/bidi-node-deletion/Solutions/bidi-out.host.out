[
 (n1, empty)
 (n3, empty)
|
]
